kubernetes/zenko/values.yaml

```diff
@@ -12,18 +12,42 @@ ingress:
   # Used to create an Ingress record.
   # This must match the 'cloudserver' 'endpoint', unless your client
   # supports different hostnames.
-  hosts:
-    - zenko.local
```

Comment:
I would not recommend removing this as it would break existing deployments when they upgrade. We can deprecate it, honor both and suggest using the new attribute. Also we need to document this, or you can use `hosts`

Reply:
I've added a fallback where if `ingress.hosts` is defined it will be respected, along with a deprecation notice in `values.yaml` and in the message printed after install/upgrade if it is being used